docs/.vuepress/theme/util/index.js

```diff
@@ -197,15 +197,21 @@ export function resolveMatchingConfig (regularPath, config) {
       config: config
     }
   }
-  for (const base in config) {
-    if (ensureEndingSlash(regularPath).indexOf(encodeURI(base)) === 0) {
+  const regularPathWithSlash = ensureEndingSlash(regularPath)
+
+  const matchingConfig = Object.entries(config)
+    .filter(([base, _]) => regularPathWithSlash.indexOf(encodeURI(base)) === 0)
+    .sort(([base, _value], [nextBase, _nextValue]) => {
+      return nextBase.length - base.length;
+    })
+    .map(([base, value]) => {
       return {
         base,
-        config: config[base]
+        config: value
       }
-    }
-  }
-  return {}
+    })[0]
```

Comment:
Non issue: 

Mapping then reading the first entry seems a little odd, but and easy to miss the `[0]` when reading. 

Maybe:

``` js
const matchingConfig = Object.entries(config)
    .map(([base, config]) => ({ base, config }))
    .sort(({ base: a }, { base: b }) => a.length - b.length)
    .find(({ base }) => regularPathWithSlash.includes(base));
```  
  
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/find
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/includes

Reply:
@George-Payne fair point, updated in https://github.com/EventStore/documentation/pull/332/commits/a8dd2cae629d1c3b43d9addfe755bbe8c22450e6